packages/c/sshnpd/include/sshnpd/params.h

```diff
@@ -21,8 +21,11 @@ struct _sshnpd_params {
   size_t manager_list_len;
   char **manager_list;
 
+  char *policy;
+
   size_t permitopen_len;
-  char **permitopen;
+  char **permitopen_hosts;
+  uint16_t *permitopen_ports; // 0 = '*'
```

Comment:
previously permitopen was a list of strings, but now it is split up into a list of hosts and list of ports, "*" for port = 0